apps/dalmatiner_frontend/src/dalmatiner_key_h.erl

```diff
@@ -21,7 +19,7 @@ handle(Req, State) ->
         html ->
             F = fun (Socket, Transport) ->
                         File = code:priv_dir(dalmatiner_frontend) ++
-                            "/static/metric.html",
+                            "/static/key.html",
                         Transport:sendfile(Socket, File)
```

Comment:
Remove the dynamic function call on line 23. Only modules that define callbacks should make dynamic calls.